src/EntityFrameworkCore.Projectables/Services/ProjectableExpressionReplacer.cs

```diff
@@ -209,19 +209,30 @@ when type.IsAssignableFrom(operand.Type)
                 if (nodeExpression is not null)
                 {
                     _expressionArgumentReplacer.ParameterArgumentMapping.Add(reflectedExpression.Parameters[0], nodeExpression);
+                    if (reflectedExpression.Parameters.Count > 1)
+                    {
+                        var projectableAttribute = nodeMember.GetCustomAttribute<ProjectableAttribute>(false)!;
+                        foreach (var prm in reflectedExpression.Parameters.Skip(1).Select((Parameter, Index) => new { Parameter, Index }))
```

Comment:
I would prefer calling this `parameterWithIndex` to stay in sync with the naming conventions used throughout the rest of the code base

Reply:
Comment accepted